data/config/toolbox.conf

```diff
@@ -0,0 +1,13 @@
+[general]
+# Default used distribution (e.g., fedora, rhel)
+# Taken from ID field in /etc/os-release
```

Comment:
This is the *fallback* distribution, in case the host OS isn't supported, right? The default should still the same distribution as the host as long as it's recognized.

We should explain the distinction better in the comment. Otherwise it can be confusing.

Reply:
> This is the fallback distribution, in case the host OS isn't supported, right?

No, I meant it as an override that supersedes even the host OS recognition. Does that sound like a bad option to have?

Reply:
Ok, I see. Thinking about this some more, what you are saying makes more sense because it makes it identical to the `--distro` option.

Generally speaking, the configuration file options should mimic the command line ones. If something can't be tweaked through the command line, then it shouldn't be possible through the configuration file either. There might be valid exceptions, but as a general guideline, this makes things more manageable (ie., easier to document, explain, etc.).

Reply:
In that same spirit, we should also use the same blurbs that we use to document the command line options in the manuals. That would reiterate the fact that they are the same options, but with a different priority (ie., command line overrides configuration file).

Any deviation has the potential to confuse a user in unexpected ways.